### PR TITLE
doc new fields for eth_getTransaction series of RPCs

### LINKED ISSUE #602: Doc new fields for eth_getTransaction series of RPCs
Dev PR: https://github.com/hyperledger/besu/pull/1835

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [x] you fixed all the issues raised by the tests, if any.
- [x] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

Add new fields to the three eth_getTransaction JSON-RPC samples.

## Issue fixed

fixes #602 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

## Testing

https://hyperledger-besu--637.org.readthedocs.build/en/637/

### CODE REVIEW

**Review by bgravenorst — CHANGES_REQUESTED** [review]
Looks like you may need to update the Transaction object table as well. See https://besu.hyperledger.org/en/latest/Reference/API-Objects/#transaction-object

**Review by NicolasMassart — COMMENTED** [review]
Does it have to be added also to graphql results?
By the way, graphql result for eth_getTransactionByBlockHashAndIndex seems strange as it has nothing to do with the RPC result and doesn't provide info about the TX. Mistake?

@hyperledger/besu-docs-maintainers ?

File: docs/Reference/API-Objects.md [hunk 1]
```diff
@@ -177,12 +177,15 @@ and
 |----------------------|:-------------------:|--------------------------------------------------------------------------------------------|
 | **blockHash**        | Data, 32&nbsp;bytes | Hash of the block containing this transaction. `null` when transaction is pending.         |
 | **blockNumber**      | Quantity            | Block number of the block containing this transaction. `null` when transaction is pending. |
+| **chainId**          | Quantity            | [Chain ID](../Concepts/NetworkID-And-ChainID.md).                                          |
```
Comment (macfarla) [inline docs/Reference/API-Objects.md:180]: I just spotted typo a few lines up `eth_getTransactionsByBlockNumberAndIndex`

should be `eth_getTransactionByBlockNumberAndIndex` - I think it's related enough to this change to fix in the same PR

**Review by macfarla — APPROVED** [review]
There's a typo (that you didn't make!) that would be good to fix as part of this PR. Otherwise LGTM

**Pull request merged**